test/automation/src/positron/positronPlots.ts

```diff
@@ -52,6 +54,10 @@ export class PositronPlots {
 		await this.code.waitForElement(CURRENT_STATIC_PLOT);
 	}
 
+	async waitForWebviewPlot(selector: string) {
+		await this.code.driver.getFrame(OUTER_WEBVIEW_FRAME).last().frameLocator(INNER_WEBVIEW_FRAME).last().locator(selector).waitFor({ state: 'visible' });
+	}
```

Comment:
Getting the last frame for inner and outer because when you clear a plot, the frame seems to stay behind.  Fortunately, new frames seem to be added sequentially.

Reply:
I think it's a bug that we're not properly disposing webview plots. Fixing that would also let us add `waitForNoPlots()` to all test cases.

Reply:
Note that Wasim has put in a fix for this.